provisioner/ansible/provisioner.go

```diff
@@ -111,6 +112,13 @@ func (p *Provisioner) Prepare(raws ...interface{}) error {
 		p.config.LocalPort = "0"
 	}
 
+	if p.config.User == "" {
+		p.config.User = os.ExpandEnv("${USER}")
```

Comment:
Curious why you used `os.ExpandEnv("${USER}")` rather than `os.Getenv("USER")` here. ExpandEnv uses Getenv under the hood -- do we need the side effects of expand for something?